packages/react-tabster/src/hooks/useFocusIndicatorStyle.ts

```diff
@@ -99,3 +99,96 @@ export const createCustomFocusIndicatorStyle = (
   },
   [`${KEYBOARD_NAV_SELECTOR} :${options.selector || defaultOptions.selector}`]: style,
 });
+
+/**
+ * @param pressedBorderColor - color used in case user presses again the field.
+ * @param targetChild - primary element if style is applied to a wrapper.
+ */
+export type BottomFocusIndicatorOptions = CreateFocusIndicatorStyleRuleOptions & {
+  borderWidth: string;
+  borderRadius: string;
+  borderColor: string;
+  pressedBorderColor: string;
+  targetChild?: keyof JSX.IntrinsicElements;
```

Comment:
should the `targetChild` really be optional ? you use it in the outline styles

Reply:
it should be optional because the outline should only be removed if the focus border is applied to a wrapper slot. I can remove this option, but I added it in case a component in the future applies the styles to the primary slot.